pegen/pegen.h

```diff
@@ -26,6 +26,11 @@ typedef struct {
     PyArena *arena;
 } Parser;
 
+typedef struct {
+    alias_ty alias;
+    int line, col, endline, endcol;
```

Comment:
Honestly I still think there will be some utility in naming these variables the same as they are in `expr_ty` (even if their offsets aren't). And ditto for Token.